.gitmodules

```diff
@@ -1,12 +1,14 @@
 [submodule "modules/native-android"]
 	path = modules/native-android
-	url = https://github.com/hashcube/native-android.git
+	url = https://github.com/CodeCombiner/native-android.git
```

Comment:
this change isn't needed

Reply:
@CodeCombiner this change also not needed

Reply:
Done

Reply:
is this change needed?

Reply:
Lets fix this at the end. If we change it now, during the transition there would be a lot of confusion. That's why I have kept this PR open.